### ISSUE TITLE
Framework Issues

### ISSUE THREAD

**Issue by amosavian** [abbeycode/UnzipKit#48]
Hi,

I wanted to add framework for dynamic linking to my project, it doesn't show iOS framework in `Embedded Binaries` and even when I fixed it, I had issues for `resources.bundle` file.

Interestingly, I had not these issues with UnrarKit, while I get `UnrarKit/Classes/UnrarKit.h:12:9: 'Cocoa/Cocoa.h' file not found` error.

**Comment by abbeycode** [comment]
Are you still having this problem?

**Comment by amosavian** [comment]
I didn't try again. Falled back to manual inclusion. I'll try and inform you

**Comment by amosavian** [comment]
@abbeycode I solved the problem with `UnrarKit` by adding new line to `UnrarKit.h` umbrella header:

```objective-c
#import <TargetConditionals.h> //  <-- THIS LINE ADDED

#if TARGET_OS_IPHONE
#import <UIKit/UIKit.h>
#else
#import <Cocoa/Cocoa.h>
#endif
```

I will tell about UnzipKit if I overcome the problem

**Comment by amosavian** [comment]
I got this error when tried to upload to iTunesConnect, using `UnrarKit.framework` in project:

    ERROR ITMS-90060: "This bundle is invalid. The value for key CFBundleShortVersionString '2.8-beta' in the Info.plist file must be a period-separated list of at most three non-negative integers."

**Comment by abbeycode** [comment]
Interesting. If you're working off of a fork, just update the `Info.plist` not to be a beta (just `2.8`).

**Comment by amosavian** [comment]
I did and now is uploading the binary \o/

**Comment by abbeycode** [comment]
How are you producing the Framework? Are you using Carthage?

**Comment by amosavian** [comment]
Nope, simply dragged `.xcodeproj` into my project and added it to `Embedded Binary` section.
Carthage automatically rectifies some common malformations and universal binaries. But I'm more convenient with this style

**Comment by abbeycode** [comment]
I was reading back through this thread, and I couldn't tell if UnzipKit is being affected, or if this only affected UnrarKit. Is there any work left to do on UnzipKit related to using it as a dynamic framework?

**Comment by amosavian** [comment]
I didn't tried to figure out how to resolve the problem with UnzipKit yet, I'll try after new year eve.

**Comment by amosavian** [comment]
I tried UnzipKit framework again, now the problem is `UnzipKit-iOS` target is not enlisted in `Embedded Binaries` section in project settings when I want to add it while I can see `UnzipKit   macOS` in the list. I tried to solve problem by changing some project settings without success.
I supposedly has to recreate iOS target from scratch.

**Comment by abbeycode** [comment]
Can you try the `issue48` branch? I think it should work for you now. If so, I'll cut a new release.

**Comment by amosavian** [comment]
@abbeycode tested and it works perfectly

**Comment by abbeycode** [comment]
This has been merged to `master` and is included in v1.8.3. Thanks again for pointing it out!

**Comment by amosavian** [comment]
Thank you very much
